ant-node/src/node.rs

```diff
@@ -618,6 +618,14 @@ impl Node {
                             }
                         }
                     }
+                    Ok(None) => {
+                        error!("Quoting metrics not found for {key:?}. This might be because we are using a ClientRecordStore??. This should not happen");
+                        QueryResponse::GetStoreQuote {
+                            quote: Err(ProtocolError::GetStoreQuoteFailed),
+                            peer_address: NetworkAddress::from_peer(self_id),
+                            storage_proofs,
+                        }
+                    }
```

Comment:
Would getting rid of this `Option` type solve the issue of the `"This should not happen"` error here?